app/models/globus_destination.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# Model representing a Globus destination that can be used as a staging location.
+class GlobusDestination < ApplicationRecord
+  belongs_to :batch_context, optional: true
```

Comment:
Is there value in relating the `GlobusDestination` and `User` models here? 

```ruby
belongs_to: user
```

That would give us model access for walking between them?

Reply:
I don't know. I noticed that a `BatchContext` belongs_to a `User`, so there is that relationship already. Would that help with what we need to connect up?

Reply:
Updated to add a foreign key for the `user` table row. It's possible to do a sunet lookup with something like `user.sunet_id` on the model, and as I do the next part with some helper methods on the model, will consider adding some a method for this if it seems useful.